README.md

```diff
@@ -173,11 +173,13 @@ In order to speed up testing, Barista keeps in mind some considerations.
 # Download
 
 ```gradle
-androidTestCompile('com.schibsted.spain:barista:1.1.0') {
+androidTestCompile('com.schibsted.spain:barista:x.y.z') {
```

Comment:
This is the contrary of automating. This is telling the user: 
"Ha! You thought you could come here and copy&paste the dependency and be happy, right? Well, you don't! Go back to the top of the page, read the version name, remember it (because you can't copy it), paste the dependency in your project, come back here because you already forgot the version, and replace the x.y.z in your project with the actual version."

That's how I feel when I find something like this.